test/v1/services/CreateAndAmendOtherExpensesServiceSpec.scala

```diff
@@ -20,26 +20,22 @@ import v1.mocks.connectors.MockCreateAndAmendOtherExpensesConnector
 import v1.models.domain.Nino
 import v1.models.errors._
 import v1.models.outcomes.ResponseWrapper
-import v1.models.request.createAndAmendOtherExpenses.{
-  CreateAndAmendOtherExpensesBody,
-  CreateAndAmendOtherExpensesRequest,
-  PatentRoyaltiesPayments,
-  PaymentsToTradeUnionsForDeathBenefits
-}
+import v1.models.request.TaxYear
+import v1.models.request.createAndAmendOtherExpenses._
 
 import scala.concurrent.Future
 
 class CreateAndAmendOtherExpensesServiceSpec extends ServiceSpec {
 
-  val taxYear    = "2017-18"
+  val taxYear    = "2021-22"
```

Comment:
can we update this spec too align with [this](https://github.com/hmrc/individuals-reliefs-api/blob/main/test/v1/services/DeleteForeignReliefsServiceSpec.scala) one? i.e. moving things into the test trait and the test to the bottom and ensuring the test wording is consistent so the tests can be read quickly when viewing the file